AppCenterCrashes/AppCenterCrashes.xcodeproj/project.pbxproj

```diff
@@ -1525,6 +1528,57 @@
 			shellPath = /bin/sh;
 			shellScript = "cd ${SRCROOT}/../Scripts/AppCenterCrashes\n./build-appcentercrashes-ios-docs.sh";
 		};
+		F8B8EADA21105BA6004BCECB /* Verify no buildSettings */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+				"$(SRCROOT)/../Config/VerifyNoBS.swift",
+			);
+			name = "Verify no buildSettings";
+			outputPaths = (
+				"$(DERIVED_FILE_DIR)/VerifyNoBS",
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "xcrun -sdk macosx swiftc -target x86_64-macosx10.9 $SCRIPT_INPUT_FILE_0 -o $SCRIPT_OUTPUT_FILE_0\n$SCRIPT_OUTPUT_FILE_0 ${PROJECT_NAME}.xcodeproj/project.pbxproj";
```

Comment:
You're probably right 😇

Reply:
Actually it can make sense, particularly when it's the same script on all targets. If we have to modify it, we'll do it in only place if the script is in a separate bash file.

Reply:
I thought some more, and I think we should have the 1-line script in one .sh file like I suggested earlier and @clpolet also agrees =)

Reply:
I've found the better solution